pkgs/top-level/perl-packages.nix

```diff
@@ -3477,6 +3477,23 @@ let
     buildInputs = [ CryptOpenSSLGuess ];
   };
 
+  CryptPBKDF2 = buildPerlPackage {
+    pname = "Crypt-PBKDF2";
+    version = "0.161520";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/A/AR/ARODLAND/Crypt-PBKDF2-0.161520.tar.gz";
+      sha256 = "97dfa79a309a086e184a4e61047f8a10ffb3db051025e7d222a25f19130ba417";
+    };
+    buildInputs = [ TestFatal ];
+    propagatedBuildInputs = [ DigestHMAC DigestSHA3 ModuleRuntime Moo TryTiny TypeTiny namespaceautoclean strictures ];
```

Comment:
`ModuleRuntime` is not needed, it is propagated from the other deps

Reply:
Fixed.